src/goo/scriptpack/ScriptComponentHandler.js

```diff
@@ -58,10 +58,10 @@ define([
 		});
 	};
 
-	ScriptComponentHandler.prototype._updateScriptInstance = function (instanceConfig, options) {
+	ScriptComponentHandler.prototype._updateScriptInstance = function (component, instanceConfig, options) {
 		var that = this;
 
-		return this._createOrLoadScript(instanceConfig)
+		return this._createOrLoadScript(component, instanceConfig)
 		.then(function (script) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/rikard/goojs/file/2299240684/issues/source?bid=3193730&fileBranchId=3193879#l65)